pkg/zfs/volume.go

```diff
@@ -157,6 +157,20 @@ func GetZFSVolume(volumeID string) (*apis.ZFSVolume, error) {
 	return vol, err
 }
 
+// GetZFSVolumeStatus returns ZFSVolume status
+func GetZFSVolumeStatus(volID string) (string, string, error) {
```

Comment:
Need to add more comments/context for this function. 
what does the OwnerID contain? Probably should rename as it is returning OwnerID or may be add another function to get the OwnerID.

Reply:
updated the comment.

Reply:
Let us rename this to GetZFSVolumeState. Status usually means online/offline. Of course, the same argument can be made for State as well. See if there is a better name, if not resolve this comment.